platform/engineering/official_github_teams_process

```diff
@@ -0,0 +1,38 @@
+# GitHub Teams on for VA.gov
```

Comment:
I think this page could have a more generic title. It's really about the authorization / user management approach for VSP.

Reply:
Oh, it's also missing a file extension :)

Reply:
Good point, and I changed to file name as well to be more generic.  Plus added a .md extension.

Reply:
To Do: Change file name as well